File: src/test/java/testdata/java/SpecialObjectTestPOJO.java
package testdata.java;

import java.math.BigDecimal;
import java.time.*;
import java.time.temporal.Temporal;
import java.util.Date;
import java.util.UUID;

public class SpecialObjectTestPOJO {


    private Byte aByte = 0;

    private Short aShort = 0;

    private Integer integer = 0;

    private Long aLong = 0L;

    private Float aFloat = 0F;

    private Double aDouble = 0D;

    private Boolean aBoolean = false;

    private Character character = 'c';

    // -----

    private String string = "";

    private BigDecimal bigDecimal = BigDecimal.ZERO;

    //-----


    private Date date = new Date();

    private Temporal temporal = Instant.now();

    private LocalDateTime localDateTime = LocalDateTime.now();

    private LocalDate localDate = LocalDate.now();

    private LocalTime localTime = LocalTime.now();

    private ZonedDateTime zonedDateTime = ZonedDateTime.now();

    private YearMonth yearMonth = YearMonth.now();

    //-----

    private UUID uuid = UUID.randomUUID();

}

File: src/test/java/testdata/kotlin/SpecialObjectTestPOJO.kt
package testdata.kotlin

import java.math.BigDecimal
import java.time.*
import java.time.temporal.Temporal
import java.util.*

class SpecialObjectTestPOJO {
    private val aByte: Byte = 0
    private val aShort: Short = 0
    private val integer = 0
    private val aLong = 0L
    private val aFloat = 0f
    private val aDouble = 0.0
    private val aBoolean = false
    private val character = 'c'

    // -----
    private val string = ""
    private val bigDecimal = BigDecimal.ZERO

    //-----
    private val date = Date()
    private val temporal: Temporal = Instant.now()
    private val localDateTime = LocalDateTime.now()
    private val localDate = LocalDate.now()
    private val localTime = LocalTime.now()
    private val zonedDateTime = ZonedDateTime.now()
    private val yearMonth = YearMonth.now()
}